catalog.html

```diff
@@ -0,0 +1,280 @@
+         </nav>
+       </header>
+
+
+
+    <main class="main-inner">
+       <header class="inner-header">
+        <h1 class="inner-header-title">Гостиницы Седоны</h1>
+    <ul class="breadcrumbs">
+        <li class="breadcrumbs-item">
+          <a class="breadcrumbs-link" href="index.html">Главная</a>
+        </li>
+
+     <li class="breadcrumbs-item">
+            <a class="breadcrumbs-link">Гостиницы</a>
+          </li>
+        </ul>
+      </header>
+
+     <section class="catalog-products">
+        <h2 class="visually-hidden">Список товаров с фильтрами</h2>
+      <form class="catalog-filter" action="https://echo.htmlacademy.ru/" method="get">
+          <fieldset class="catalog-filter-group">
+            <legend class="catalog-filter-title">Инфраструктура:</legend>
+
+            <ul class="catalog-filter-list">
+            <li class="catalog-filter-item">
+                <label class="control">
+                  <input class="control-input" type="checkbox" name="Бассейн">
+                  Бассейн
+                </label>
+              </li>
+
+              <li class="catalog-filter-item">
+                <label class="control">
+                  <input class="control-input" type="checkbox" name="Парковка" checked>
+                  Парковка
+                </label>
+              </li>
+
+              <li class="catalog-filter-item">
+                <label class="control">
+                  <input class="control-input" type="checkbox" name="Wi-Fi">
+                  Wi-Fi
+                </label>
+              </li>
+            </ul>
+          </fieldset>
+
+          <fieldset class="catalog-filter-group">
+            <legend class="catalog-filter-title">Тип Жилья:</legend>
+            <ul class="catalog-filter-list">
+
+
+            <li class="catalog-filter-item">
+                <label class="control">
+                  <input class="control-input" type="radio" name="Гостиница">
+                  Гостиница
+              </label>
+              </li>
+
+            <li class="catalog-filter-item">
+                <label class="control">
+                  <input class="control-input" type="radio" name="Мотель">
+                  Мотель
+                </label>
+              </li>
+
+            <li class="catalog-filter-item">
+                <label class="control">
+                  <input class="control-input" type="radio" name="Апартаменты">
+                  Апартаменты
+              </label>
+               </li>
+              </ul>
+          </fieldset>
+
+        </form>
+      </section>
+
+      <form action="#" method="get">
+        <fieldset>
+          <legend>Стоимость, <span>₽</span></legend>
+          <label>от<input type="text" value="0"></label>
+          <label>до<input type="text" value="1000"></label>
+        </fieldset>
+
+        <button type="button" name="popup-button">Применить</button>
+        <button type="button" name="popup-button">Сбросить</button>

```

Comment:
Должен быть тип `reset`:
```suggestion
        <button type="reset" name="popup-button">Сбросить</button>
```